src/main/java/de/tum/in/www1/artemis/web/rest/settings/ide/IdeMappingDTO.java

```diff
@@ -0,0 +1,19 @@
+package de.tum.in.www1.artemis.web.rest.settings.ide;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.enumeration.ProgrammingLanguage;
+import de.tum.in.www1.artemis.domain.settings.ide.Ide;
+
+/**
+ *
+ * @param programmingLanguage
+ * @param ide
+ */
```

Comment:
I'd say we should either describe these parameters & the record or leave out the doc comment